### DISCUSSION TITLE
Incompatibility of IDPKit with new SSI Web-Wallet Stack

### DISCUSSION THREAD

**Discussion by vanhoanHoang** [walt-id/.github#72] [Help] [answered]
Hello team, 

I have been testing the new WaltID-Web-Wallet stack with the latest version of OIDC protcols that you have implemented. However, the stack seems to be imcompatible with the latest version of IDPKit. 

Do you have a plan to upgrade the IDPKit please ?

**Comment by taminobaumann** [comment, marked as answer]
Hey @vanhoanHoang, 

thanks for reaching out. Yes, at the moment the IDP-Kit is unfortunately incompatible with the new stack. At this point, our focus is on launching the main components for [the Community Stack](https://walt.id/blog/p/community-stack). After that, we move on to the higher level components such as the IDP-Kit. Therefore, I cannot give you a concrete date for the upgrade of the IDP-Kit yet. Though, it won't be this year.

Best
Tamino
